apps/client/src/app/modules/list/list-details-panel/list-details-panel.component.html

```diff
@@ -101,12 +101,18 @@
                   nzSize="small">
             <i class="material-icons">local_atm</i>
           </button>
-          <button (cbOnSuccess)="textCopied()" (click)="$event.stopPropagation()" [cbContent]="getTextExport()"
+          <button (click)="$event.stopPropagation();copyTextExport()"
```

Comment:
So I thought a bit more about this clipboard issue and I think we should have our own lazy clipboard directive, instead of having to import the service in every single component that does clipboard copy stuff.